lib/pact_broker/webhooks/url_redactor.rb

```diff
@@ -0,0 +1,49 @@
+module PactBroker
+  module Webhooks
+    class URLRedactor
+      # @url = attributes[:url]
+      PARAMS_TO_REDACT = [/auth/i, /token/i]
+      SLACK_URL = "hooks.slack.com/services/"
+      attr_accessor :url
```

Comment:
All the methods are class methods - can you explain what the attr_accessor is for? That's typically used for instance variables, which a class does not have.

Reply:
I was struggling to get access to the contents of url, when calling the class from the `webhook_request.rb`, and after some googling, that is what I came back with, but as stated, I am not using it in the correct context. i'll take it out and see what happens!

Out of interest, when we drop into each of the if statements that will modify the url, should we try the operations, and if they fail, just revert back to the original url? I want to try and avoid any application errors as a result of an unhandled errors (I am used to try/catch blocks in Javascript)

Thanks!

Reply:
removed now 👍